src/parse.js

```diff
@@ -12,8 +13,9 @@ const STATE_MODIFIER = Symbol("modifier");
 const STATE_FUNCTION = Symbol("function");
 const STATE_NAME = Symbol("name");
 
-export function parseCell(input, {globals} = {}) {
-  const cell = CellParser.parse(input);
+export function parseCell(input, {globals, tag = {}} = {}) {
+  const source = tag.name ? toJavaScript(input, tag.name, tag.raw) : input;
```

Comment:
How about flattening the options? And also I’d probably use a nullish check here instead of falsey.

```suggestion
export function parseCell(input, {globals, tag, raw} = {}) {
  const source = tag != null ? toJavaScript(input, tag, raw) : input;
```